packages/json-schema-worker/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export default {
+	async fetch(request: Request, env: Env): Promise<Response> {
+		const url = new URL(request.url);
+		const matches = url.pathname.match(/\/(.*?)\.json/);
```

Comment:
Is this `json-schema-worker` exclusively for testing? Or is the idea that we'll serve a production version from here too? If it's just the latter, I reckon you could probably just re-use the `prerelease-registry` and save yourself the maintenance of this new service.